satellite/console/consoleweb/consoleapi/analytics.go

```diff
@@ -56,16 +56,22 @@ func (a *Analytics) EventTriggered(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		a.serveJSONError(w, http.StatusInternalServerError, err)
 	}
+
 	userID, err := a.service.GetUserID(ctx)
 	if err != nil {
 		a.serveJSONError(w, http.StatusInternalServerError, err)
 		return
 	}
 
+	user, err := a.service.GetUser(ctx, userID)
```

Comment:
GetUser will actually make a request from the databse - this should not be necessary. If you use `console.GetAuth(ctx)` above instead of `a.service.GetUserID(ctx)`, I think you can use the return value to get `auth.User.ID` and `auth.User.Email`

for example, see `GetAccount(` in satellite/console/consoleweb/consoleapi/auth.go